### ISSUE TITLE
Localisation of ResetPasswordNotification 

### ISSUE THREAD

**Issue by mauthi** [laravel-enso/Core#55]
This is  a bug.

### Description
Subject / Text is not translated

```
return (new MailMessage())
            ->line('Please set or reset your password by clicking the button below.')
            ->action('Direct Link', config('app.url').'/password/reset/'.$this->token)
            ->line('Thank you for using our application!');
```

### Steps to Reproduce
Set `config.app.locale` to something different to en
Go to login page
Go to reset password
Enter a valid email address
start `php artisan queue:work`

### Expected behavior
Mailtext should be translated

### Actual behavior
Mailtext is in english only

**Comment by aocneanu** [comment]
@mauthi, Why don't you just make a PR and add the `__` ?

**Comment by mauthi** [comment]
> because I didn’t have time at the moment I saw this :)
will do it

**Comment by aocneanu** [comment]
Np. I believe that you can also assign yourself to open issues, right? :)

**Comment by mauthi** [comment]
Translation of rest of the mail not possible at the moment - see https://github.com/laravel/ideas/issues/409

**Comment by mauthi** [comment]
closed with #58
